app/test_split_integer.py

```diff
@@ -2,20 +2,26 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
 
+    assert split_integer(17, 4) == [4, 4, 4, 5]
```

Comment:
are you checking the sum here?

Reply:
Yes, but i have some similar tests, so i just gonna remove this one